ui/src/components/NavigationBar/NavBar.js

```diff
@@ -22,17 +22,23 @@ import {
   Link,
   LinkOverlay,
   useDisclosure,
+  // Spinner,
```

Comment:
can remove the stuff related to `isLoading` like the `Spinner` and other things you commented out if it's not necessary